packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -270,37 +292,6 @@
             grid-gap: 0.8rem 0.4rem;
             flex: 1;
             padding: 0.4rem 0;
-            .dc-contract-card {
```

Comment:
@hamza-deriv please restore this. it's needed for other contracts.

Reply:
this class is only needed in the multipliers and if i add this again it will break the css of the multipliers in the Mobile View. I removed it because this css was needed for recent-positions card view, so i added it separately by extending it in --turbos class.

Reply:
As it is only applicable on mobile so issued it inside the mobile mixin

`&-recent-positions {
                display: grid;
                grid-template-columns: 1fr 1fr 1fr;
                grid-template-areas:
                    'stake current-stake barrier-level'
                    'profit-loss limit-order-info';
                grid-gap: 0.8rem 0.4rem;
                flex: 1;
                padding: 0.4rem 0;
                .dc-contract-card {
                    &__stake {
                        grid-column: 1/1;
                        grid-row: 1/1;
                        padding: 0.8rem 0;
                    }
                    &__current-price {
                        grid-column: 2/2;
                        grid-row: 1/2;
                        padding: 0.8rem 0;
                    }
                    &__buy-price {
                        grid-column: 1/2;
                        grid-row: 2/2;
                    }
                    &__barrier-level {
                        grid-column: 2/2;
                        grid-row: 2/2;
                    }
                    &__limit-order-info {
                        grid-row: 2/2;
                        grid-column: 2/2;
                        position: relative;
                    }
                    &__status {
                        align-self: center;
                        justify-self: center;
                        grid-row: 1/3;
                        grid-column: 3/3;
                    }
                }
            }`

Reply:
removed this identifier and added as main css in --turbos